### PR TITLE
Picard

### PR DESCRIPTION
**Pull request by jonca79**

### CODE REVIEW

File: workflow/rules/picard_hs_metrics.smk [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "qc/picard_hs_metrics/{sample}_{type}.log",
+    benchmark:
+        repeat(
+            "qc/picard_hs_metrics/{sample}_{type}.benchmark.tsv", config.get("picard_hs_metrics", {}).get("benchmark_repeats", 1)
+        )
+    threads: config.get("picard_hs_metrics", {}).get("threads", config["default_resources"]["threads"])
+    resources:
+        threads=config.get("picard_hs_metrics", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("picard_hs_metrics", {}).get("time", config["default_resources"]["time"]),
+        mem_mb=config.get("picard_hs_metrics", {}).get("mem_mb", 1024),
```
Comment (Smeds) [inline workflow/rules/picard_hs_metrics.smk:30]: should we have something for this in default_resources? @jonca79 @marrip

Comment (marrip) [reply workflow/rules/picard_hs_metrics.smk:30]: Definitely!

Comment (jonca79) [reply workflow/rules/picard_hs_metrics.smk]: Memory now added to default

**Review by Smeds — APPROVED** [review]
We should make sure that the other rules and module uses mem_mb

**Pull request merged**